.stylelintrc

```diff
@@ -10,6 +10,7 @@
     "at-rule-no-unknown": null,
     "declaration-property-unit-whitelist": null,
     "function-url-quotes": "always",
+    "selector-id-pattern": null,
```

Comment:
Same here, let's not globally ignore these types of issues because they should get flagged. In this case, all problems seem to be caused by `#google_dashboard_widget` which however is generated by WordPress. So let's instead annotate these specific lines to be ignore by stylelint.

Reply:
Ok, updated.